pom.xml

```diff
@@ -437,6 +437,71 @@
               <modelNameSuffix />
             </configuration>
           </execution>
+          <!-- OPEN API CLIENT TEMPLATE-ENGINE -->
+          <execution>
+            <phase>generate-resources</phase>
+            <id>generate-client-templates-engine</id>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <inputSpec>
+                https://raw.githubusercontent.com/pagopa/pn-templates-engine/refs/heads/develop/docs/openapi/pn-internal-templates-v1.yaml
+              </inputSpec>
+              <skipValidateSpec>true</skipValidateSpec>
+              <generatorName>java</generatorName>
+              <library>resttemplate</library>
+              <generateApiTests>false</generateApiTests>
+              <generateModelTests>false</generateModelTests>
+              <typeMappings>
+                <typeMapping>OffsetDateTime=java.time.Instant</typeMapping>
+                <typeMapping>file=byte[]</typeMapping>
+              </typeMappings>
+              <templateDirectory>${project.build.directory}/dependency-resources/scripts/openapi/templates/6.0.0/client</templateDirectory>
+              <configOptions>
+                <apiPackage>${project.groupId}.user.attributes.generated.openapi.msclient.templatesengine.api</apiPackage>
+                <modelPackage>${project.groupId}.user.attributes.generated.openapi.msclient.templatesengine.model</modelPackage>
+                <dateLibrary>java8</dateLibrary>
+                <annotationLibrary>none</annotationLibrary>
+                <documentationProvider>source</documentationProvider>
+                <openApiNullable>false</openApiNullable>
+                <useAbstractionForFiles>true</useAbstractionForFiles>
+                <useBeanValidation>false</useBeanValidation>
+              </configOptions>
+            </configuration>
+          </execution>
+          <!-- OPEN API CLIENT TEMPLATE-ENGINE REACTIVE-->
```

Comment:
Serve il client reactive?

Reply:
non serve, ma l'abbiamo lasciato in linea come su altri ms